src/components/Publish/_validation.ts

```diff
@@ -103,7 +103,7 @@ const validationPricing = {
     .max(10, 'Maximum is 10%')
     .test(
       'maxDigitsAfterDecimal',
-      `Must have maximum ${MAX_DECIMALS - 1} decimal digits`,
+      `Must have maximum ${MAX_DECIMALS - 2} decimal digits`,
```

Comment:
same as above

Reply:
Publishing asset doesn't work for more than 4 decimals in case of the swap fee field

Reply:
ah, you are right, there is a min value